src/brevitas/graph/target/flexml.py

```diff
@@ -45,6 +45,12 @@
             'weight_quant': Int8WeightPerTensorFixedPoint,
             'bias_quant': Int16Bias,
             'return_quant_tensor': True}),
+    nn.Conv3d: (
+        qnn.QuantConv3d,
+        {
+            'weight_quant': Int8WeightPerTensorFixedPoint,
+            'bias_quant': Int16Bias,
+            'return_quant_tensor': True}),
```

Comment:
@Giuseppe5, do we need Conv3d for the flexml target?

Reply:
No

Reply:
shall I remove this part?

Reply:
Yeah